index.html

```diff
@@ -17,7 +17,13 @@
     {% for dev in site.data.contributions.devs %}
       <div class="col-sm-2 mb-4">
         <div class="card h-100">
-          <a href="#"><img class="card-img-top" src="{{ dev.avatar }}" alt=""></a>
+          <a href="#">
+          {% if dev.avatar %}
```

Comment:
Well, this will not resolve the issue, you need to use the onerror attribue on img tag to handle if url provided it invalid.